scripting/include/multi1v1.inc

```diff
@@ -4,6 +4,8 @@
 #endif
 #define _multi1v1_included
 
+#define DISTRIBUTION_SPREAD 1000.0
```

Comment:
I think I'd favor prefixing these with a Multi1v1_ prefix for external consumption. The constants below should have been in the past, but that ship has sailed.